apps/web/app/modules/Admin/EditCourse/compontents/CourseCardPreview.tsx

```diff
@@ -0,0 +1,54 @@
+import DefaultCoursePhoto from "~/assets/default-photo-course.svg";
+import { Button } from "~/components/ui/button";
+import { Card, CardContent, CardHeader } from "~/components/ui/card";
+import { UserProfile } from "~/modules/Dashboard/DashboardNavigation/UserProfile";
+
+type CourseCardPreviewProps = {
+  imageUrl?: string;
+  title?: string;
+  category?: string;
+  description?: string;
+};
+
+const CourseCardPreview = ({ imageUrl, title, description, category }: CourseCardPreviewProps) => {
+  return (
+    <Card className="p-6 shadow-md border border-gray-200 w-3/10">
+      <CardHeader className="relative">
+        <h5 className="text-xl font-semibold">Card Preview</h5>
+      </CardHeader>
+      <CardContent>
+        <p>This is how your Course will be visible to students.</p>
+        <div className="flex justify-center">
+          <Card className="mt-4 w-full max-w-xs rounded-lg shadow-md relative">
+            <CardHeader className="p-0 rounded-t-lg overflow-hidden">
+              <img
+                src={imageUrl || DefaultCoursePhoto}
+                alt="header"
+                className="w-full h-[15rem] object-cover"
```

Comment:
Same case as above 😉 
```suggestion
                className="w-full h-60 object-cover"
```